models/events.sql

```diff
@@ -184,6 +184,11 @@ select
                     "path": "$.user_properties",
                     "dtype": "object",
                 },
+                {
+                    "name": "event_properties",
+                    "path": "$.event_properties",
```

Comment:
```suggestion
                    "path": "$",
```

We can use $ to refer to the entire object. The string `$.event_properties` is saying "event_properties" inside "event_properties" which doesn't exist.

Reply:
Ah I see! Thanks for explaining that!